cli/flox/src/commands/install.rs

```diff
@@ -221,6 +219,8 @@ impl Install {
             Err(EnvironmentSelectError::Anyhow(e)) => Err(e)?,
             Err(e) => Err(e)?,
         };
+        environment_subcommand_metric!("install", concrete_environment);
```

Comment:
We may miss out on a few commands that would error out between lines 122-222. Maybe I should add a subcommand_metric at the top?

Reply:
Nonblocking, but yeah, I'd prefer if we recorded that right at the top.

Reply:
Okay I added everywhere it isn't just turning an EnvironmentSelect or a ServicesEnvironment